pkg/cmd/roachtest/hibernate_blacklist.go

```diff
@@ -23,8 +23,8 @@ var hibernateBlacklists = blacklistsForVersion{
 // After a failed run, an updated version of this blacklist should be available
 // in the test log.
 var hibernateBlackList20_1 = blacklist{
-	"org.hibernate.id.QuotedIdentifierTest.testDirectIdPropertyAccess":                                                                                                               "26925",
-	"org.hibernate.id.hhh12973.PostgreSQLSequenceGeneratorWithSerialTest.test":                                                                                                       "unknown",
+	"org.hibernate.id.QuotedIdentifierTest.testDirectIdPropertyAccess":                                                                                                               "24062",
+	"org.hibernate.id.hhh12973.PostgreSQLSequenceGeneratorWithSerialTest.test":                                                                                                       "24062",
```

Comment:
This issue is closed - shoudl we open a new and more appropriate one?

Reply:
yeah, I created this but forgot to use it here: https://github.com/cockroachdb/cockroach/issues/42662